publications-2019.bib

```diff
@@ -1,5 +1,13 @@
 % Encoding: US-ASCII
 
+@phdthesis{Bzowski2019,
+  author       = {Krzysztof Bzowski}, 
```

Comment:
Congratulations :-)

Reply:
Thank you.  Without deal.ii it would be much harder.